test/helpers/test_capturing.rb

```diff
@@ -101,6 +101,32 @@ def test_different_sites
     assert_equal 'Content Two', content_for(@item, :b)
   end
 
+  def test_content_for_duplicate_symbols
+    with_site do |_site|
+      # Prepare
+      File.open('lib/helpers.rb', 'w') do |io|
+        io.write 'include Nanoc::Helpers::Capturing'
+      end
+      File.open('content/includer.erb', 'w') do |io|
+        io.write '[<%= content_for(@items.find { |i| i.identifier == \'/includee/\' }, :blah) %>]'
```

Comment:
Ok, I changed that, but note that all other tests in that file use the other style